integrations/mediawiki.js

```diff
@@ -0,0 +1,178 @@
+const util = require('util');
+const fetch = require('node-fetch');
+const groupArray = require('group-array');
+const cron = require('node-cron');
+
+module.exports = async function(robot, kredits) {
+
+  function messageRoom(message) {
+    robot.messageRoom(process.env.KREDITS_ROOM, message);
+  }
+
+  robot.logger.debug('[hubot-kredits] Loading MediaWiki integration...')
+
+  const Contributor = kredits.Contributor;
+  const Operator = kredits.Operator;
+
+  const apiURL =  process.env.KREDITS_MEDIAWIKI_URL + 'api.php';
+
+  function getContributorByWikiUser(username) {
+    return Contributor.all().then(contributors => {
+      let contrib = contributors.find(c => {
+        if (typeof c.accounts !== 'object') { return false; }
+        return c.accounts.find(a => {
+          a.url === `${process.env.KREDITS_MEDIAWIKI_URL}User:${username}`;
+        });
+      });
+      if (!contrib) {
+        throw new Error();
```

Comment:
could provider an error message here, something like `contributor not found ${username}`